docs/src/pages/demos/tabs/FullWidthTabs.js

```diff
@@ -58,15 +60,17 @@ class FullWidthTabs extends React.Component {
             <Tab label="Item Three" />
           </Tabs>
         </AppBar>
-        <SwipeableViews
-          axis={theme.direction === 'rtl' ? 'x-reverse' : 'x'}
-          index={this.state.value}
-          onChangeIndex={this.handleChangeIndex}
-        >
-          <TabContainer dir={theme.direction}>Item One</TabContainer>
-          <TabContainer dir={theme.direction}>Item Two</TabContainer>
-          <TabContainer dir={theme.direction}>Item Three</TabContainer>
-        </SwipeableViews>
+        <Typography type="body1">
```

Comment:
I would rather put it in the `TabContainer` component.

Reply:
I considered that, but thought it would result in muliple additional components being mounted all applying the same style. I'll go with your lead on it.

Out of curiosity, is there a reason we can't set this at the top level as the default Typography?

Reply:
Not much more effort if you replace the `div` with a `<Typography component="div">`.